website/content/api-docs/system/config-reload.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: api
+page_title: /sys/config/reload - HTTP API
+description: The '/sys/config/reload' endpoint is used to reload specific parts of Vault's configuration.
+---
+
+# `/sys/config/reload`
+
+The `sys/config/reload` endpoint allows reloading specific parts of Vault's configuration.
+Currently, it only supports reloading license information from files on disk.
+
+| Method | Path                          |
+| :----- | :---------------------------- |
+| `GET`  | `/sys/config/reload/:subsystem` |
```

Comment:
It's a POST, not a GET.

Reply:
It's actually a `PUT` now that I look at it, but good point - I'll update.